api_tests/src/actors/actor.ts

```diff
@@ -237,7 +237,7 @@ export class Actor {
         };
         this.betaAsset = {
             name: AssetKind.Erc20,
-            quantity: create.beta.amount,
+            quantity: create.alpha.amount,
```

Comment:
That seems like a bug no? The field this is assigned to is named `this.betaAsset`, why are we assigning the `alpha` amount to it?